src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-11-1-upgrade-notes.md

```diff
@@ -79,5 +85,81 @@ Removed variables can be safely deleted from your themes or elements. Deprecated
+
+### Label deprecation
+
+Since [WEBUI-116](https://jira.nuxeo.com/browse/WEBUI-116), the `document history` and `audit page` use the same component and the new prefix `audit`. The following are some examples of the deprecated labels and their replacement:
+
+| Deprecated labels             | New Labels          |
+|-------------------------------|---------------------|
+| documentHistory.category      | audit.category      |
+| documentHistory.comment       | audit.comment       |
+| documentHistory.date          | audit.date          |
+| documentHistory.filter.after  | audit.filter.after  |
+| documentHistory.filter.before | audit.filter.before |
+
+### Reference to invalid packages in the project
+
+The reference to projects that no longer exist, such as `nuxeo-dam` or `nuxeo-spreadsheet`, or the presence of JSF specific contributions, might cause conflicts and prevent a project from working properly.
+
+#### Nuxeo DAM addon removal
+
+Nuxeo DAM no longer exists as an addon and its contributions are now default on Web UI.
+
+#### Spreadsheet addon removal
+
+Spreadsheet addon is now loaded by default but the button contribution is disabled. Users can rely on Studio Designer to re-enable the `spreadSheet` *button* in the `RESULTS_ACTIONS` slot (see [WEBUI-90](https://jira.nuxeo.com/browse/WEBUI-90)).
+
+### BROWSER_ACTIONS slot removal
+
+The `BROWSER_ACTIONS` nuxeo slot was removed under [NXP-26184](https://jira.nuxeo.com/browse/NXP-26184). It was already deprecated since Web UI 0.9 and had no known usage. It was replaced by the `RESULTS_SELECTION_ACTIONS` slot.
+
+### Forked nuxeo-document-content might lose selection actions
+
+After [NXP-25345](https://jira.nuxeo.com/browse/NXP-25345), Nuxeo Web UI introduced the ability to override selection actions. Elements that where forked from an older version of `nuxeo-document-content` and that override the `selectionActions` native slot with new content will be missing the contributions to the `RESULTS_SELECTION_ACTIONS` nuxeo slot. This can be rectified by adding the desired actions to the new slot, and by deleting the following piece of code:
+
+```
+<div slot=“selectionActions”>
+  <nuxeo-slot slot=“BROWSE_ACTIONS” model=“[[browseActionContext]]“></nuxeo-slot>
+</div>
+```
+
+### Picture document page being displayed on doctypes using the Picture facet
+
+Since [NXP-25740](https://jira.nuxeo.com/browse/NXP-25740), the `nuxeo-picture-document-page` is now displayed for documents with **facet** `Picture` instead of documents with **type** `Picture`. This means that on migrated projects, the `nuxeo-picture-document-page` might be displayed on documents where it was not expected. If this is not desirable, the contribution can simply be overridden to only display the page for documents with **type** `Picture`:
```

Comment:
```suggestion
Since [NXP-25740](https://jira.nuxeo.com/browse/NXP-25740), the `nuxeo-picture-document-page` is now displayed for documents with the `Picture` **facet** instead of documents from the `Picture` **type**. This means that on migrated projects, the `nuxeo-picture-document-page` might be displayed on documents where it was not expected. If this is not desirable, the contribution can simply be overridden to only display the page for documents with the `Picture` **type**:
```

Reply:
👍 Thanks